### PR TITLE
HomeScreen

### PR DESCRIPTION
**Pull request by ZafeerurRahim**
The home screen would have navigation that is the buttons work navigating the user to their desired place.

### CODE REVIEW

File: QuizzingApp341 - Backup.csproj
Comment (benderje26) [inline QuizzingApp341 - Backup.csproj:1]: I don't think this backup file should be here

File: QuizzingApp341.csproj
Comment (benderje26) [inline QuizzingApp341.csproj:1]: And this probably should stay as views, not pages, I don't know if either of these files should've gotten changed except for the things that have to do with TestQuiz.xaml and TestQuiz.xaml.cs. The rest just seems like it is things that went wrong with downloading these from git

File: Views/HomeScreen.xaml.cs
Comment (benderje26) [inline Views/HomeScreen.xaml.cs:1]: looks good

File: Views/TestQuiz.xaml
Comment (benderje26) [inline Views/TestQuiz.xaml:1]: looks good

File: Views/TestQuiz.xaml.cs
Comment (benderje26) [inline Views/TestQuiz.xaml.cs:1]: looks good

**Review by benderje26 — CHANGES_REQUESTED** [review]
The things with "QuizzingApp341 - Backup.csproj" and "QuizzingApp341.csprog" need to be addressed, it might be as simple as deleting QuizzingApp341.csproj and then renaming the backup to QuizzingApp341.csproj, not sure.

**Review by skogmanpe40 — APPROVED** [review]
Everything ran properly for me, so approving pull request so we can try and get main up to date with all the code so we can get everything done for tomorrow

**Review by benderje26 — APPROVED** [review]
Looks good other than adding supabase-csharp but I'll just remove that next  commit

**Pull request merged**